File: sdk/javascript-closure-angular/API/Client/RouteDetail.js
goog.provide('API.Client.RouteDetail');

/**
 * @record
 */
API.Client.RouteDetail = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetail.prototype.operator;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetail.prototype.origin;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetail.prototype.originlocalized;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetail.prototype.destination;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetail.prototype.destinationlocalized;

/**
 * @type {!string}
 * @export
 */
API.Client.RouteDetail.prototype.lastupdated;

/**
 * @type {!Array<!API.Client.Stop>}
 * @export
 */
API.Client.RouteDetail.prototype.stops;

File: sdk/scala/src/main/scala/io/swagger/client/model/RouteDetail.scala
/**
 * RTPI REST Web Services API
 * <p>RTPI REST Web Services API (<a href=\"https://goo.gl/s6NTJe\">RRWS API</a>) provides REST interface to retrieve information on real time bus information, timetables and bus stops. </p> <p>Each method available in RRWS API will be invoked using plain HTTP Request (GET method). Depending on parameters provided, result will be returned either in XML or JSON format (if format is not provided then JSON is used by default). Response will contain requested information (see method specification for details) and error status of the request.</p>  <h2>Error Codes</h2> <table> <tr><th>Error Code</th><th>Description</th></tr> <tr><td>0</td><td>Success</td></tr> <tr><td>1</td><td>No Results</td></tr> <tr><td>2</td><td>Missing Parameter</td></tr> <tr><td>3</td><td>Invalid Parameter</td></tr> <tr><td>4</td><td>Sceduled Downtime</td></tr> <tr><td>5</td><td>Unexpected System Error</td></tr> </table> <p>Error code is included in response for each message. For some error codes additional details will be provided in free text format (Error Message field).</p> 
 *
 * OpenAPI spec version: 1.0.0
 * 
 *
 * NOTE: This class is auto generated by the swagger code generator program.
 * https://github.com/swagger-api/swagger-codegen.git
 * Do not edit the class manually.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package io.swagger.client.model




case class RouteDetail (
  operator: String,
origin: String,
originlocalized: String,
destination: String,
destinationlocalized: String,
lastupdated: String,
stops: List[Stop])
